src/quo2/components/wallet/account_overview/view.cljs

```diff
@@ -0,0 +1,123 @@
+                                                     (colors/custom-color :blue 60)
+                                                     theme)
+        currency-change-color   (colors/theme-colors (colors/custom-color :blue 50)
+                                                     (colors/custom-color :blue 60)
+                                                     theme)
+        dot-separator-color     (colors/theme-colors (colors/custom-color :blue 60 40)
+                                                     (colors/custom-color :blue 50 40)
+                                                     theme)
+        loading-state-color     (colors/theme-colors colors/neutral-5 colors/neutral-90 theme)]
+    [rn/view
+     {:style style/account-overview-wrapper}
+     (if loading?
+       [loading-state loading-state-color]
+       [:<>
+        [rn/view {:style {:flex-direction :row}}
+         [quo2/text
+          {:weight :medium
+           :size   :paragraph-1
+           :style  (style/account-name account-name-color)}
+          account-name]
+         (when (= account :watched-address)
+           [icons/icon :i/reveal
+            {:container-style style/reveal-icon
+             :color           reveal-icon-color}])]
+        [quo2/text
+         {:weight :semi-bold
+          :size   :heading-1
+          :style  style/current-value}
+         current-value]
+        [rn/view
+         {:style style/row-centered}
+         [:<>
+          [quo2/text
+           {:weight :medium
+            :size   :paragraph-2
+            :style  (style/bottom-time-text (and (not= :custom time-frame)
+                                                 (seq time-frame-to-string)))}
+           (when (seq time-frame-string)
+             time-frame-string)]
+          (when (and (= :custom time-frame)
+                     (seq time-frame-to-string))
+            [:<>
+             [icons/icon :i/positive-right
+              {:size            16
+               :container-style style/right-arrow}]
+             [quo2/text
+              {:weight :medium
+               :size   :paragraph-2
+               :style  style/bottom-time-to-text}
+              time-frame-to-string]])]
+         (when (and (seq percentage-change)
+                    (seq currency-change))
+           [:<>
+            [quo2/text
+             {:weight :medium
+              :size   :paragraph-2
+              :style  (style/percentage-change percentage-change-color)}
+             percentage-change]
+            [rn/view
+             {:style (style/dot-separator dot-separator-color)}]
+            [quo2/text
+             {:weight :medium
+              :size   :paragraph-2
+              :style  (style/currency-change currency-change-color)}
+             currency-change]
+            [icons/icon
+             (if up?
+               :i/positive
+               :i/negative)
+             {:color positive-icon-color
+              :size  16}]])]])]))
+
+(def account-overview
```

Comment:
btw aligning with the best practices doc we should export this as `view`. So we can use `view` and `view-internal` here 👍

Reply:
Done